src/Pages/NoticePage/style.ts

```diff
@@ -0,0 +1,135 @@
+import styled from "styled-components";
+
+export const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #000000;
+  height: 100vh;
+`;
+
+export const FilterContainer = styled.header`
+  width: 50rem;
+  padding: 0, 1.25rem;
+  display: flex;
+  margin-top: 2.5rem;
+  position: relative;
+
+  & > div {
+    position: absolute;
+    top: 4rem;
+    z-index: 10;
+    left: 1.25rem;
+  }
+`;
+
+export const FilterButton = styled.button`
+  background-color: #292424;
+  border-radius: 3.125rem;
+  font-size: 1.125rem;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 120%;
+  color: #f3a4b2;
+  display: flex;
+  align-items: center;
+  column-gap: 0.5rem;
+  border: none;
+  padding: 0.625rem 1rem;
+  cursor: pointer;
+  margin-left: 1.25rem;
+
+  &:hover {
+    box-shadow: 0 0 18px 0px #f3a4b2;
+  }
+`;
+
+export const CloseButton = styled.button`
+  margin-left: 1.25rem;
+  border: none;
+  border-radius: 3.125rem;
+  cursor: pointer;
+  width: 6rem;
+  height: 2.75rem;
+  background-color: #f3a4b2;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+export const ListContainer = styled.ul`
+  margin-top: 2.69rem;
+  width: 50rem;
+  padding: 0, 1.25rem;
+  display: flex;
+  flex-direction: column;
+  row-gap: 1.75rem;
+  align-items: center;
+`;
+
+export const ListItem = styled.li<{
+  requestType: string;
+  isOnlyOne: boolean;
+}>`
```

Comment:
```suggestion
export const ListItem = styled.li`
```

사용하지 않는 스타일은 제거해주세요 ~